File: src/main/java/com/crl/bettercommandline/BetterCommandLine.java
package com.crl.bettercommandline;

import com.crl.bettercommandline.config.ModConfigManager;
import net.fabricmc.api.ModInitializer;

public class BetterCommandLine implements ModInitializer {
    public static final String MOD_ID = "bettercommandline";

    @Override
    public void onInitialize() {
        ModConfigManager.initializeConfig();
        HistoryManager.load();
    }
}

File: src/main/kotlin/com/crl/bettercommandline/BetterCommandLine.kt
package com.crl.bettercommandline

import com.crl.bettercommandline.config.ModConfigManager
import net.fabricmc.api.ModInitializer

class BetterCommandLine : ModInitializer {
    companion object {
        const val MOD_ID = "bettercommandline"
    }

    override fun onInitialize() {
        ModConfigManager.initializeConfig()
        HistoryManager.load()
    }
}